dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -117,6 +117,7 @@ add-tag = Add Tag
 Add-Template = Add Template
 add-template = Add Template
 Add-To-Bundle = Add To Bundle
+Add-All-Categories-To-Bundle = Add all categories to bundle
```

Comment:
Capital C and B. "Add all Categories to Bundle"

Reply:
done https://github.com/dotCMS/core/pull/23183/commits/f391675dc6195856bc48cfd178fd3253b1177121